README.md

```diff
@@ -22,6 +22,7 @@ Although most configuration is contained in this repository, some external depen
       - `DESEC_NS_APIKEY`: `ns` API key needed for replication operations
       - `DESEC_NS_CARBONSERVER`: pdns `carbon-server` setting (optional)
       - `DESEC_NS_CARBONOURNAME`: pdns `carbon-ourname` setting (optional)
+      - `DESEC_NS_NAME`: name under which this name server will be reached, e.g. ns1.example.com
```

Comment:
Couldn't this be optional, too (just like the signing key)?

Reply:
it is; README is clarified